module/herosystem6e.mjs

```diff
@@ -209,7 +224,22 @@ Hooks.once("devModeReady", ({ registerPackageDebugFlag }) => {
 export class HEROSYS {
     static ID = "HEROSYS";
 
-    static module = "hero6efoundryvttv2";
+    static onInitialise() {
+        console.log("HEROSYS running - no system load issues");
+        return null;
+    }
+
+    static module = HEROSYS.onInitialise();
```

Comment:
Unclear how this ends up being hero6efoundryvttv2

Reply:
That was so I get a log statement when the module is correct enough to load. The actual value is not written there until the `fetchWithTimeout` promise completes. 

The trouble I had was with static initializers refusing to be initialized with asynchronous loads.
If there are examples that you have seen that grab the data in a static fashion that would make things less complex, and I would choose them.

I tried `require` and `import` as my first attempts. I may not have had the syntax correct, and so if you've used those to pull in a json file that would also be preferable. 

I couldn't get them to work.

The only way I could get at that system.json was with some kind of fetch, or load, 

I chose to use the foundry function because it involved a library we already had at our fingertips and there was no need to import anything else.

I resolved the timing of the rest of the initialization by wrapping all the stuff that 'goes first' and kicks off the rest of the system in the `then` that comes after the json is decoded and the __actual__ HEROSYS.module is given the content of sytem.json.id .

that happens here:
https://github.com/coderodent-calfee/hero6e-foundryvtt/blob/coderodent/gamesystem-2/module/herosystem6e.mjs#L47

Reply:
ok turns out the info needed was present in game.system.id when the 'init' hook is called. 
I removed the async load in (https://github.com/dmdorman/hero6e-foundryvtt/pull/938/commits/7720053af8ae49a411aae0392ade85ab61758bd2) and made sure it still works.

Simpler is better,